packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -193,7 +230,7 @@ export type TAppLinks = {
 export const LinksMapper: Record<TPlatforms.All, TAppLinks> = {
     ctrader: {
         android: 'https://play.google.com/store/apps/details?id=com.deriv.ct',
-        ios: 'https://apps.apple.com/cy/app/ctrader/id767428811',
+        ios: 'https://apps.apple.com/us/app/deriv-ctrader',
```

Comment:
Just for confirmation, @hirad-deriv did we verify if this link without the id still works?

Reply:
nope @shahzaib-deriv the appid is a unique identifier used by playstore to go specifically to the app, without the id it will just go to the app search / landing page